chalice/app.py

```diff
@@ -0,0 +1,147 @@
+import json
+import traceback
+
+from chalice import Chalice, NotFoundError, BadRequestError
+from cloud_blobstore import BlobNotFoundError
+from chalicelib.config import logger
+from chalicelib.matrix_handler import LoomMatrixHandler
+from chalicelib.request_handler import RequestHandler, RequestStatus
+from chalicelib.sqs import SqsQueueHandler
+
+app = Chalice(app_name='matrix-service-api')
+app.debug = True
+
+# Replace handler here for supporting concatenation on other matrix formats
+mtx_handler = LoomMatrixHandler()
+
+
+@app.route('/matrices/health', methods=['GET'])
+def health():
+    return {'status': 'OK'}
+
+
+@app.route('/matrices/concat/{request_id}', methods=['GET'])
+def check_request_status(request_id):
+    """
+    Check the status of a matrices concatenation request based on
+    a request ID.
+
+    :param request_id: Matrices concatenation request ID.
+    """
+    try:
+        merge_request_body = RequestHandler.get_request_body(request_id=request_id)
+        return merge_request_body
+
+    except BlobNotFoundError:
+        raise NotFoundError("Request({}) has not been initialized.".format(request_id))
+
+    except:
```

Comment:
out of the scope of this diff, but it might make sense to mark all your routes with a decorator that serves as a common fallback exception handler.  this way, you can produce error messages with a consistent format.

Reply:
Thanks for this good advice. I will do that.